convert_fendl.py

```diff
@@ -41,90 +42,308 @@ class CustomFormatter(argparse.ArgumentDefaultsHelpFormatter,
+# This can be extended to accommodate new releases
 release_details = {
     '3.1a': {
-        'base_url': 'https://www-nds.iaea.org/fendl31/data/neutron/',
-        'files': ['fendl31a-neutron-ace.zip'],
-        'neutron_files': ace_files_dir.glob('*'),
-        'compressed_file_size': '0.4 GB',
-        'uncompressed_file_size': '3 GB'
+        'neutron':{
+            'base_url': 'https://www-nds.iaea.org/fendl31/data/neutron/',
+            'files': ['fendl31a-neutron-ace.zip'],
+            'file_type': 'ace',
+            'ace_files': ace_files_dir.glob('*.ace'),
+            'compressed_file_size': 384,
+            'uncompressed_file_size': 2250
+        },
+        'photon':{
+            'base_url': 'https://www-nds.iaea.org/fendl31/data/atom/',
+            'files': ['fendl30-atom-endf.zip'],
+            'file_type': 'endf',
+            'photo_files': endf_files_dir.joinpath('endf').glob('*.txt'),
+            'compressed_file_size': 4,
+            'uncompressed_file_size': 12
+        }
     },
     '3.1d': {
-        'base_url': 'https://www-nds.iaea.org/fendl/data/neutron/',
-        'files': ['fendl31d-neutron-ace.zip'],
-        'neutron_files': ace_files_dir.joinpath('fendl31d_ACE').glob('*'),
-        'compressed_file_size': '0.5 GB',
-        'uncompressed_file_size': '3 GB'
+        'neutron':{
+            'base_url': 'https://www-nds.iaea.org/fendl/data/neutron/',
+            'files': ['fendl31d-neutron-ace.zip'],
+            'file_type': 'ace',
+            'ace_files': ace_files_dir.joinpath('fendl31d_ACE').glob('*'),
+            'compressed_file_size': 425,
+            'uncompressed_file_size': 2290
+        },
+        'photon':{
+            'base_url': 'https://www-nds.iaea.org/fendl/data/atom/',
+            'files': ['fendl30-atom-endf.zip'],
+            'file_type': 'endf',
+            'photo_files': endf_files_dir.joinpath('endf').glob('*.txt'),
+            'compressed_file_size': 4,
+            'uncompressed_file_size': 12
+        }
+    },
+    '3.0': {
+        'neutron':{
+            'base_url': 'https://www-nds.iaea.org/fendl30/data/neutron/',
+            'files': ['fendl30-neutron-ace.zip'],
+            'file_type': 'ace',
+            'ace_files': ace_files_dir.joinpath('ace').glob('*.ace'),
+            'compressed_file_size': 364,
+            'uncompressed_file_size': 2200
+        },
+        'photon':{
+            'base_url': 'https://www-nds.iaea.org/fendl30/data/atom/',
+            'files': ['fendl30-atom-endf.zip'],
+            'file_type': 'endf',
+            'photo_files': endf_files_dir.joinpath('endf').glob('*.txt'),
+            'compressed_file_size': 4,
+            'uncompressed_file_size': 12
+        }
+    },
+    '2.1': {
+        'neutron':{
+            'base_url': 'https://www-nds.iaea.org/fendl21/fendl21mc/',
+            'files': ['H001mc.zip',  'H002mc.zip',  'H003mc.zip',  'He003mc.zip', 
```

Comment:
Such a shame FENDL didn't release a single zip file for this release, but great to see the openmc scripts are flexible to accommodate such an input.